editor/src/components/canvas/canvas-strategies/strategies/grid-element-change-location-strategy.spec.browser2.tsx

```diff
@@ -594,9 +699,9 @@ export var storyboard = (
             child.style
           expect({ top, left, gridColumnStart, gridColumnEnd, gridRowStart, gridRowEnd }).toEqual({
             gridColumnStart: '1',
-            gridColumnEnd: '',
+            gridColumnEnd: 'auto',
```

Comment:
reminder: these are NOT being serialized, it's just that the computed style defaults these to auto if a shorthand is used (which is fine)